src/startup.lisp

```diff
@@ -23,9 +23,17 @@
 
 (in-package "EMOTIQ")
 
+(defun directorify (str)
```

Comment:
The right way to do this is to convert to a `cl:pathname` rather than relying on namestring semantics:  you can't open a file without a valid `cl:pathname`